src/lib/AddOverlay.svelte

```diff
@@ -1,48 +1,164 @@
 <script lang="ts">
-	import { addOverlayVisible } from '../stores';
-	import ActionButton from './ActionButton.svelte';
+	import { addOverlayVisible, infoOverlayVisible, infoOverlayActiveTab } from '../stores';
 	import CloseButton from './CloseButton.svelte';
+	import { SvelteToast, toast } from '@zerodevx/svelte-toast';
+	import { onMount } from 'svelte';
+
+	const app = new SvelteToast({
```

Comment:
I think this is not needed. Removing it doesn't seem to change anything. Then also remove the import.